packages/docs/developer-resources/integrations/listings.md

```diff
@@ -0,0 +1,123 @@
+* You are going to be running your own Celo blockchain node (either on your own hardware or on a cloud provider infrastructure).
+* You want to list CELO (previously cGLD)  and cUSD on your exchange or ranking site and you want to find useful information that is relevant for listing those digital assets.
+* You want to find relevant resources for brand assets that will visually aid in those listings.
+
+## Support
+If you have any questions or need assistance with these instructions, please contact cLabs or ask in the #exchanges channel on [Celo’s Discord server](https://chat.celo.org/) . Remember that Discord is a public channel: never disclose recovery phrases (also known as backup keys, or mnemonics), private keys, unsanitized log output, or personal information.
+
+This guide will also help you find all the necessary information about brand assets, how to integrate with Celo and what useful listing information are made available to you as well as any information about looking for support.
+
+The Celo Protocol Github is located [here.](https://github.com/celo-org/)
+
+
+## Outline
+In this guide you will:
+* Go through an audit checklist to ensure top security requirements are met prior to running your node.
+* Run a Celo blockchain network node.
+* Find the relevant brand assets for the Celo protocol.
+* How to integrate Celo protocol into your infrastructure.
+* Learn about the name change from cGLD to CELO.
+* Learn about useful API endpoints for your listing needs.
+* Go over the NPM documentation for your listing needs.
+* Learn about the Stability protocol and whitepaper.
+
+## Audits
+All the security audits on the smart contracts, security and economics of the Celo Platform can be found [here](https://celo.org/audits).
+
+## Running a Celo Node
+In order to get started with the Celo Platform for your listing needs, you need to be able to run a Celo full-node. This helps you audit the distributed ledger by having your own node running. The documentation on running a full-node are found [here](https://docs.celo.org/getting-started/mainnet/running-a-full-node-in-mainnet).
+
+## Celo Brand Assets for Listing
+If you are listing Celo on your exchange, you will probably need access to the Celo Platform brand assets. They can be found [here](https://celo.org/experience/brand#overview).  
+
+Please ensure your use of the Celo Platform assets provided follows our brand policy found [here](https://celo.org/brand-policy).
+
+## Integrating Celo With Your Infrastructure
+There are several ways to integrate the Celo Platform with your infrastructure.
+
+A general overview of integrations that would be relevant to you listing Celo Platform are shown [here](https://docs.celo.org/developer-guide/overview/integrations/general).
+
+For more specific use-cases for exchanges, please checkout the [Custody Integration Guide](https://docs.celo.org/developer-guide/overview/integrations/custody) as well.
```

Comment:
maybe we should rename this section as well?

Reply:
I was thinking of that as well